packages/input/src/input-group.tsx

```diff
@@ -75,10 +45,26 @@ export const InputGroup = forwardRef<InputGroupProps, "div">((props, ref) => {
         width: "100%",
         display: "flex",
         position: "relative",
+        ".chakra-input__left-addon ~ .chakra-input": {
+          borderLeftRadius: leftAddonBorderLeftRadius ?? 0,
```

Comment:
instead of `?? 0` we could assign the default when destructuring the props above:

```js
  const {
    children,
    className,
    leftElementWidth,
    rightElementWidth,
    leftAddonBorderLeftRadius = 0,
    righAddonBorderRightRadius,
    ...rest
  } = omitThemingProps(props)
  ```